app.rb

```diff
@@ -18,7 +20,7 @@ def run
 
```

Comment:
![image](https://github.com/sergio-usma/ruby_school_library/assets/93039359/cabaa173-47c5-4709-a7cd-4688e80a76fd)

**[OPTIONAL]**

- [ ] It would be nice if you make the Ruby a link to the Ruby official website to make your readme more descriptive and professional 😊